backend/de.metas.async/src/main/sql/postgresql/system/80-async-dlm/5624381_sys_gh12591_migrate_c_queue_package_processor_ID.sql

```diff
@@ -1,3 +1,68 @@
+
+
+-- BACKUP
+
+CREATE TABLE backup.BKP_c_queue_block_15023023 AS
+SELECT *
+FROM c_queue_block
+;
+
+CREATE TABLE backup.BKP_c_queue_workpackage_15032023 AS
+SELECT *
+FROM c_queue_workpackage
+;
+
+CREATE TABLE backup.BKP_dlm_c_queue_workpackage_archived_15032023 AS
+SELECT *
+FROM dlm.c_queue_workpackage_archived
+;
+
+CREATE TABLE backup.BKP_c_queue_element_15032023 AS
+SELECT *
+FROM c_queue_element
+;
+
+
+-- DELETE invalid data
+
+CREATE TEMP TABLE workPackageIDsToDelete AS
```

Comment:
It's fine like this, but in general i would consider to not have this as a temp table. 
Instead create the table as `backup.workPackageIDsToDelete_15032023`.
Then you can later see what was deleted and maybe x-check with a backup or archive data.